config/sidekiq_scheduler.yml

```diff
@@ -2,3 +2,7 @@ CreateDailySpoolFiles:
   cron: "0 3 * * * America/New_York"
   class: EducationForm::CreateDailySpoolFiles
   description: "Generate a daily set of spool files and transmit them over SFTP to a backend system"
+
+SidekiqStatsJob:
+  every: 1m
```

Comment:
Nice catch - yes, it was running multiple times per minute. Tested locally after update to use `cron` option and job is only executed once across all workers.